certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificate/service/XmlSchematronValidator.java

```diff
@@ -0,0 +1,11 @@
+package se.inera.intyg.certificateservice.domain.certificate.service;
+
+import se.inera.intyg.certificateservice.domain.certificate.model.CertificateId;
+import se.inera.intyg.certificateservice.domain.certificate.model.Xml;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.SchematronPath;
+
+public interface XmlSchematronValidator {
```

Comment:
Move this interface to clinicalprocess gradle module as its not necessary to expose it as part of the domain.